src/components/onboarding/steps.js

```diff
@@ -218,13 +218,13 @@ export const steps = t => ([
     position: 'right',
     style: styles.step,
   },
-  {
-    title: t('Access extra features'),
-    text: t('Change account settings, enable delegate voting and repeat the onboarding here.'),
-    selector: '.more-menu',
-    position: 'right',
-    style: styles.step,
-  },
+  // {
+  //   title: t('Access extra features'),
+  //   text: t('Change account settings, enable delegate voting and repeat the onboarding here.'),
+  //   selector: '.more-menu',
+  //   position: 'right',
+  //   style: styles.step,
+  // },
```

Comment:
Instead of disabling this step, I would prefer to just change the selector to `selector: '#settings',`. As also done in https://github.com/LiskHQ/lisk-hub/pull/856/commits/03d98ecf8716b6999cd9900ef7c51e0e15acef57#diff-21bfbadb5c72d356dbdafa6e6eae5a78 . That will preserve the feature and prevent merge conflicts.